ycms/cms/templates/ward/ward_management.html

```diff
@@ -0,0 +1,87 @@
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Doctors" %}</div>
+                    <div class="text-3xl font-bold">{{ doctors_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Nurses" %}</div>
+                    <div class="text-3xl font-bold">{{ nurses_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ beds_count }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Available Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ available_beds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Occupied Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ occupied_beds }}</div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Add a New Ward" %}</div>
+            <br>
+            <form enctype="multipart/form-data"
+                  method="post"
+                  id="ward_form"
+                  class="mb-12 flex justify-between items-center gap-4"
+                  data-unsaved-warning>
+                {% csrf_token %}
+                {% for field in ward_form %}
+                    <div class="grow">
+                        <label for="{{ field.id_for_label }}"
+                               {% if field.field.required %}class="field-required"{% endif %}>{{ field.label }}</label>
+                        {% render_field field %}
+                        <div class="help-text">{{ field.help_text }}</div>
+                    </div>
+                {% endfor %}
+                <button type="submit" class="btn mt-4">{% translate "Add New Ward" %}</button>
+            </form>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Details about Wards" %}</div>
+            <table>
+                <tr>
+                    <th align="left">Ward Number</th>
+                    <th align="left">Ward Name</th>
+                    <th align="left">Number of Beds</th>
+                    <th align="left">Occupied</th>
+                    <th align="left">Available</th>
```

Comment:
```suggestion
                    <th align="left">{% translate "Ward Number" %}</th>
                    <th align="left">{% translate "Ward Name"  %}</th>
                    <th align="left">{% translate "Number of Beds"  %}</th>
                    <th align="left">{% translate "Available"  %}</th>
                    <th align="left">{% translate "Occupied"  %}</th>
```
Missing translations, and also I just noticed the order of available/occupied was reversed compared to the table rows ^^